install/libvips.js

```diff
@@ -21,7 +21,8 @@ const minimumGlibcVersionByArch = {
 };
 
 const { minimumLibvipsVersion, minimumLibvipsVersionLabelled } = libvips;
-const distBaseUrl = process.env.npm_config_sharp_dist_base_url || process.env.SHARP_DIST_BASE_URL || `https://github.com/lovell/sharp-libvips/releases/download/v${minimumLibvipsVersionLabelled}/`;
+const distHost = process.env.npm_config_sharp_libvips_binary_host || process.env.npm_config_sharp_libvips_binary_host_mirror || `https://github.com/lovell/sharp-libvips/releases/`;
+const distBaseUrl = process.env.npm_config_sharp_dist_base_url || process.env.SHARP_DIST_BASE_URL || distHost + `/download/v${minimumLibvipsVersionLabelled}/`;
```

Comment:
The `/download` part of the URL should come from the `distHost` variable.

Reply:
Yes, of course. Got a little distracted/confused by the URL from taobao.
Fix is coming!